BraveRewardsUI/Wallet/WalletViewController.swift

```diff
@@ -224,7 +224,7 @@ class WalletViewController: UIViewController, RewardsSummaryProtocol {
       self.state.delegate?.loadNewTabWithURL(url)
     }
     
-    walletView.rewardsSummaryView?.disclaimerView?.labels.forEach {
```

Comment:
`walletView.rewardsSummaryView` is nil by the time we get to this line.. It later on does `walletView.rewardsSummaryView = self.rewardsSummaryView` somewhere else in an update function. So that's the reason this fix works and will always work.